source/config/configSpec.py

```diff
@@ -187,6 +187,7 @@
 [UIA]
 	enabled = boolean(default=true)
 	useInMSWordWhenAvailable = boolean(default=false)
+	consoleUIA = boolean(default=false)

```

Comment:
```suggestion
	useInConsoleWhenAvailable = boolean(default=false)
```

Reply:
This setting is only temporary and "use when available" is misleading (as that isn't what it does). The final PR will change the setting to something like:

    winConsoleImplementation= option("auto", "legacy", "UIA", default="auto")